packages/bff/.env

```diff
@@ -32,10 +32,14 @@ GENERATED_JWT_SECONDS_DURATION=60
 S3_CUSTOM_SERVER=true
 S3_SERVER_HOST=http://localhost
 S3_SERVER_PORT=9000
+S3_BUCKET=interop-local-bucket
```

Comment:
One last comment, that could also be addressed directly by @sandrotaje in #940:

I know it's probably always the same container but in Scala there are different vars for all the different containers: one for eservice documents, one for agreement consumer documents, etc. so that they can be distinguished in the config if needed.

I would remove this S3_BUCKET var and use ESERVICE_DOCUMENTS_CONTAINER

![image](https://github.com/user-attachments/assets/e751245e-8496-48af-a5c7-b07b267fc65f)

Reply:
You can ignore :) Me an @sandrotaje agreed to track it as a further task because it is not strictly related to import/export: https://pagopa.atlassian.net/browse/IMN-813